src/app/client/src/app/modules/public/components/public-collection-player/public-collection-player.component.html

```diff
@@ -40,7 +40,7 @@ <h5 appTelemetryInteract [telemetryInteractObject]="telemetryInteractObject" [te
   <div class="row">
     <div class="one wide column"></div>
     <div class="ten wide column">
-      <app-collection-tree (contentSelect)="OnPlayContent($event)" [nodes]="collectionTreeNodes" [options]="collectionTreeOptions"></app-collection-tree>
+      <app-collection-tree (contentSelect)="OnPlayContent($event, false)" [nodes]="collectionTreeNodes" [options]="collectionTreeOptions"></app-collection-tree>
```

Comment:
please remove the false value that you don't have to send them so that even null value will work

Reply:
done